src/Microsoft.AspNetCore.Razor.Language/Legacy/CSharpCodeParser.cs

```diff
@@ -2093,5 +2246,18 @@ private static string GetName(CSharpSymbol sym)
                 return sym.Content;
             }
         }
+
+        internal class ParsedDirective
+        {
+            public string DirectiveText { get; set; }
+
+            public SourceLocation DirectiveStart { get; set; }
+
+            public string AssemblyName { get; set; }
+
+            public string TypePattern { get; set; }
+
+            public SourceLocation AssemblyNameLocation { get; set; }
```

Comment:
This is not really used in any product code but is useful in tests to make sure we parsed the correct values.